Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.7)
+    rack (2.0.8)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.2.0)
-      loofah (~> 2.2, >= 2.2.2)
-    railties (6.0.0)
-      actionpack (= 6.0.0)
-      activesupport (= 6.0.0)
+    rails-html-sanitizer (1.3.0)
+      loofah (~> 2.3)
+    railties (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
-    rake (12.3.3)
-    thor (0.20.3)
+    rake (13.0.1)
+    thor (1.0.1)
     thread_safe (0.3.6)
     tzinfo (1.2.5)
       thread_safe (~> 0.1)
-    zeitwerk (2.1.9)
+    zeitwerk (2.2.2)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.1.9 → 2.2.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **45** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.1.9...v2.2.2).

These are the first 10 commits:
- (6c9443a) [fixes signature of real_mod_name](https://github.com/fxn/zeitwerk/commit/6c9443a4cb37486dd8cbf41083a03fea476d1766)
- (757dd73) [do not mention preloads in the README](https://github.com/fxn/zeitwerk/commit/757dd734340024e487bd9678dcf80a567064bfdc)
- (47a94db) [Better error for invalid constant names](https://github.com/fxn/zeitwerk/commit/47a94db58e129409e2d181e214fe413a46d15bb3)
- (a6f1a18) [explicit begin/rescue & tweaks](https://github.com/fxn/zeitwerk/commit/a6f1a182962e9007169ef5dd74afb13abfa3c533)
- (4857328) [Merge branch 'Shopify-handle-invalid-constant-names'](https://github.com/fxn/zeitwerk/commit/4857328bfde075a2f040e9b422359b2cb0edbd25)
- (d35312f) [version 2.1.10](https://github.com/fxn/zeitwerk/commit/d35312f7355304a7c4aed34aa6e2b2e6011c0313)
- (8c4aadb) [explains why setup before main module definition](https://github.com/fxn/zeitwerk/commit/8c4aadb822b59c2170e114ec06791dd426f2ecdf)
- (1b0d316) [adds a remark about eager loading and gems](https://github.com/fxn/zeitwerk/commit/1b0d3167535e1b4b60019b778a5baafec096fb8d)
- (585569c) [documents a way to extract custom inflectors for gems](https://github.com/fxn/zeitwerk/commit/585569cd8e345edc3676e32dda6b201b185e00b2)
- (8c4c7ba) [temporarily allow ruby-head to fail in CI](https://github.com/fxn/zeitwerk/commit/8c4c7ba7075ebd9e451ca2184b2bfc6b106f565c)


</details>